backlogger.py

```diff
@@ -70,9 +83,31 @@ def issue_reminder(conf, poo):
     if "comment" in conf:
         msg = conf["comment"]
     if data["reminder-comment-on-issues"]:
-        if reminder_exists(conf, poo, msg):
+        journals = retrieve_journals(poo)
+        if journals is None:
+            sys.stderr.write(
+                "API for {} returned None, skipping reminder".format(poo["id"]))
+            return
+        elif reminder_exists(poo, journals):
             print("Skipping reminder for {}: a similar reminder already exists".format(poo["id"]))
+            # Low priority never change
+            if priority == "Low" and has_repeat_reminder:
+                print("Skip priority update for {}".format(poo["id"]))
+                return
+            if has_repeat_reminder and (last_reminder + slo_priorities[priority]["period"]) < present:
+                # second action: automatic priority
+                msg = "Automatic degrade priority from {} to {} for {}"
+                print(msg.format(priority,
+                                 slo_priorities[priority]["next_priority"]["name"],
+                                 poo["id"]))
+                url = "{}/{}.json".format(data["web"], poo["id"])
+                json_rest("PUT", url,
+                          {"issue":
+                           {"priority_id": slo_priorities[priority]["next_priority"]["id"],
+                            "notes": "Priority degraded to {}".format(
+                                slo_priorities[priority]["next_priority"]["name"])}})
             return
+        # first action: reminder
```

Comment:
I think instead of adding a comment here you want the above section to be extracted into a function. That should make the code clearer to read